### PR TITLE
Add types for ml-classify-text

### PR DESCRIPTION
**Pull request by promise**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
@promise this module looks to be UMD (when consuming by clients):
https://github.com/andreekeberg/ml-classify-text-js/blob/master/webpack.config.js#L13
the global script consumption seems to be problematic (only named exports are set directly on global scope?), so at least CommonJS module syntax should be supported out of the box (with ESM syntax via esMduleInterop set in clients tsconfig)

**Comment by promise** [comment]
@peterblazejewicz I don't fully understand what you mean, should I do any changes to the PR?

File: types/ml-classify-text/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    get confidence(): number;
+    set confidence(confidence: number);
+}
+
+export {
+    Model,
+    Vocabulary as Cocabulary, // typo by the package author
+    Classifier,
+    Classifier as default,
+};
```
Comment (peterblazejewicz) [inline types/ml-classify-text/index.d.ts:68]:
```suggestion
export { Model, Vocabulary as Cocabulary, Classifier, Classifier as default };
```
(via `prettier` used on DT)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@promise thanks!

**Comment by promise** [comment]
Ready to merge

**Comment by promise** [comment]
this is very pog

**Pull request merged**